integrationtests/esdtTransfer_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build integration
```

Comment:
might rename to `integrationtests` as `integration` could sound like a production build or something similar

Reply:
done.